src/content/developers/docs/data-availability.md

```diff
@@ -0,0 +1,153 @@
+
+Implementing data sharding requires creating a system for nodes to verify data availability in other shards without downloading blocks. If validators must download data dumped on every shard, then the purpose of sharding is defeated. 
+
+## Types of data availability systems {#types-of-data-availability-systems} 
+
+### On-chain data availability {#on-chain-data-availability} 
+
+The standard solution to solving data availability is to force block producers to publish all transaction data and have validating nodes download it. Ethereum uses this method, as do rollups (and other scaling solutions) that post transaction data on the L1 blockchain. 
+
+The problem is that on-chain data availability places bottlenecks on scalability. Full nodes have to download the same data and check it, which reduces the number of transactions that can be processed per second.
+
+More importantly, on-chain DA has negative implications for decentralization since it increases the data that full nodes must hold. If Ethereum’s state spirals, validators must invest in more expensive machines—which would probably reduce the number of people willing to run a validating node.  
+
+### Off-chain data availability {#off-chain-data-availability} 
+
+Off-chain data availability solutions move data storage off Ethereum's base layer. The block producer doesn't publish the data on-chain, but provides a cryptographic commitment to prove the availability of the data. 
+
+This is a method used by some scaling solutions, such as [validiums](/developers/docs/scaling/validium/), to reduce the amount of data posted on Ethereum. The drawback is that this system requires trusting block producers to release the data if necessary.  
+
+## What are some solutions to the data availability problem? {#solutions-to-data-availability-problem}
+
+As mentioned, the data availability problem concerns the ability to verify availability of the transaction data for a newly proposed block. Solutions to this problem employ some mechanism for guaranteeing data availability. 
+
+### Data availability committees {#data-availability-committees}
+
+Pure validiums store transaction data off-chain with a block producer, making them centralized to an extent. This reduces decentralization and security, since the block producer can publish invalid transactions and conceal the rollup's true state by hiding transaction data. 
+
+Some validiums mitigate this problem by forcing block producers to store transaction data with trusted parties that form the Data Availability Committee (DAC). Members of the DAC are expected to securely store the data off-chain and publish it on-chain in event of a dispute. They also publish attestations on-chain to prove that the said data is indeed available. 
```

Comment:
> Some validiums mitigate this problem by forcing block producers

I don't think mitigate is the correct phrasing here. (I'd personally argue that this strategy is wishful thinking in many cases)

Reply:
Changed the sentence to read "...validiums attempt to solve this problem..". And yes, DACs have risks, which I explained briefly in the intro to the following section (PoS-based DACs).